### DISCUSSION TITLE
V3 GA to npm?

### DISCUSSION THREAD

**Discussion by ryanblock** [aws/aws-sdk-js-v3#1848] [General]
Hey folks, congrats on going GA!

Just wanted to clarify: nothing in the announcement today mentioned release schedule for npm (https://aws.amazon.com/blogs/developer/modular-aws-sdk-for-javascript-is-now-generally-available/), ~when will https://www.npmjs.com/package/aws-sdk be bumped to v3?~

Update: Just noticed y’all are sticking with the `@aws-sdk` namespacing, so perhaps this question isn’t relevant!

**Comment by trivikr** [comment]
Yup, we've switched package name from `aws-sdk` to `@aws-sdk/*` while moving from v2 to v3.
